### PR TITLE
Fix flaky test on Appveyor

### LINKED ISSUE #495: Test failure in test_datetime_from_timestamp on master
The python3.6 Appveyor build on master currently has one test failure,
https://ci.appveyor.com/project/fafhrd91/pyo3/builds/24816612/job/embc7f2ccf0barfj

```py
================================== FAILURES ===================================
________________________ test_datetime_from_timestamp _________________________
    @given(dt=st.datetimes(MIN_DATETIME_FROM_TIMESTAMP,
>                          MAX_DATETIME_FROM_TIMESTAMP))
    def test_datetime_from_timestamp(dt):
tests\test_datetime.py:243: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
dt = datetime.datetime(1970, 1, 1, 0, 0)
    @given(dt=st.datetimes(MIN_DATETIME_FROM_TIMESTAMP,
                           MAX_DATETIME_FROM_TIMESTAMP))
    def test_datetime_from_timestamp(dt):
        if PYPY and dt < pdt.datetime(1900, 1, 1):
            pytest.xfail("get_timestamp will raise on PyPy with dates before 1900")
    
>       ts = get_timestamp(dt)
E       OSError: [Errno 22] Invalid argument
tests\test_datetime.py:248: OSError
--------------------------------- Hypothesis ----------------------------------
Falsifying example: test_datetime_from_timestamp(dt=datetime.datetime(1970, 1, 1, 0, 0))
```
this started happening after https://github.com/PyO3/pyo3/pull/447 was merged though the CI in this PR itself was fine. I don't know maybe this was triggered by a new `hypothesis` release changing examples that are tested.

### PR DESCRIPTION
**Pull request by Alexander-N**
Increase `MIN_DATETIME_FROM_TIMESTAMP` so that the test passes on all platforms. Maybe relevant: https://bugs.python.org/issue29097

Closes #495

~~With falsifying example from hypothesis.~~

~~e.g. in https://ci.appveyor.com/project/fafhrd91/pyo3/builds/25971133/job/5ptauuuuai8msiog~~

### CODE REVIEW

File: examples/rustapi_module/tests/test_datetime.py [hunk 1]
```diff
@@ -248,6 +248,10 @@ def test_datetime_from_timestamp(dt):
     ts = get_timestamp(dt)
     assert rdt.datetime_from_timestamp(ts) == pdt.datetime.fromtimestamp(ts)
 
+def test_datetime_from_timestamp_with_falsifying_example():
```
Comment (pganssle) [inline examples/rustapi_module/tests/test_datetime.py]: I don't think this needs to be a separate test, you can just use the `example` decorator with the existing test:

```python
@given(...)
@example(dt=datetime(1970, 1, 1, 0, 0))
def datetime_from_timestamp(dt):
    ...
```

Comment (Alexander-N) [reply examples/rustapi_module/tests/test_datetime.py]: Sure, that's nicer. Do you have any idea what the problem could be?

Comment (pganssle) [reply examples/rustapi_module/tests/test_datetime.py]: Hard to tell without a Windows computer. Timestamp support is OS-defined, so I am not terribly surprised that Windows is failing but not Linux. It does seem weird that it's failing with Python 3.6 and not 3.5, and that it only just started failing (I would be pretty surprised if `1970-01-01T00:00:00` is an unusual value for `hypothesis` to pass for datetimes).

Comment (Alexander-N) [reply examples/rustapi_module/tests/test_datetime.py]: Not sure when it started failing, but this one is from two months ago: https://ci.appveyor.com/project/fafhrd91/pyo3/builds/24579560/job/b6eddxm0jdjgl7qs

**Comment by konstin** [comment]
Thank you!

**Pull request merged**